src/style.css

```diff
@@ -1,3 +1,87 @@
+/* * {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+} */
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial, Helvetica;
+  display: flex;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  /* width: 100%; */
+  margin-bottom: 20px;
+  width: 1024px;
+}
+
+.logo_png {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+
+  /* position: relative; */
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  justify-content: center;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  justify-content: center;
+  margin-right: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  height: 15px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  font-size: 12px;
+  height: 15px;
+  color: #000;
+  justify-content: center;
+  align-items: center;
+
+  /* display: flex; */
+}
+
+a.is-active {
+  color: var(--main-color);
+}
+
+.nav__list :hover {
+  color: var(--main-color);
+}
+
+a.is-active::after {
+  content: '';
+  position: relative;
+  display: block;
+  bottom: -17px;
+  width: 100%; /* Ширина лінії дорівнює ширині тексту */
```

Comment:
```suggestion
  width: 100%;
```

Reply:
i did all changes and everything broke at all :)))